stdlib/planner/group_max_test.flux

```diff
@@ -141,32 +139,36 @@ testcase group_max_window {
 }
 testcase group_max_agg_window {
     result = testing.loadStorage(csv: input)
-        |> range(start: 2018-05-22T19:53:26Z)
+        |> range(start: 2018-05-22T19:53:26Z, stop: 2018-05-24T00:00:00Z)
         |> group(columns: ["host"])
-        |> aggregateWindow(fn: max, every: 1d)
+        |> aggregateWindow(fn: max, every: 1d, createEmpty: false)
     out_max_agg_window = "
 #datatype,string,long,dateTime:RFC3339,dateTime:RFC3339,dateTime:RFC3339,string,string,string,double
 #group,false,false,true,true,false,false,true,false,false
 #default,_result,,,,,,,,
 ,result,table,_start,_stop,_time,_measurement,host,_field,_value
-,,0,2018-05-22T19:53:26Z,2030-01-01T00:00:00Z,2018-05-23T00:00:00Z,system,host.local,load1,1.83
+,,0,2018-05-22T19:53:26Z,2018-05-24T00:00:00Z,2018-05-23T00:00:00Z,system,host.local,load1,1.83
```

Comment:
I'll leave this to your judgment, but I think it would be helpful to see a test case with multiple rows in the `want` output for when when we set `createEmpty` to `false`. Something like what you have in the `group_max_agg_window_empty` testcase, but omitting all the would-be empty rows. Thoughts?